zcashrpc-typegen/src/tokenize.rs

```diff
@@ -0,0 +1,145 @@
+
+use crate::error;
+use crate::special_cases;
+use error::TypegenResult;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+fn handle_terminal_enum(
+    label: &str,
+    name: &str,
+    called_by_alias: bool,
+) -> TokenStream {
+    let variants = label
+        .strip_prefix("ENUM:")
+        .unwrap()
+        .split(',')
+        .map(|x| x.trim());
+    let variant_idents = variants
+        .clone()
+        .map(|x| {
+            proc_macro2::TokenTree::Ident(crate::callsite_ident(
+                &x.split('-')
+                    .map(crate::capitalize_first_char)
+                    .collect::<String>(),
+            ))
+            .into()
+        })
+        .collect::<Vec<TokenStream>>();
+    let variant_idents_renames = variants
+        .map(|x| format!("#[serde(rename = \"{}\")]", x).parse().unwrap())
+        .collect::<Vec<TokenStream>>();
+    #[rustfmt::skip]
```

Comment:
Hmm.. Is this really necessary?

Reply:
I think the default formatting of 
```
      let name_tokens = crate::callsite_ident(                                    
          &(if called_by_alias {                                                  
              format!("{}Response", name)                                         
          } else {                                                                
              name.to_string()                                                    
          }),                                                                     
      );
```
Is fine

Reply:
That's not the default.   That's the custom formatting that is enabled by the skip.